r_missing_values/r_missing_values.md

```diff
@@ -0,0 +1,472 @@
+<!--
+
+author:   Rose Hartman
+email:    hartmanr1@chop.edu
+version:  0.0.1
+module_template_version: 2.0.0
+language: en
+narrator: UK English Female
+title: Missing Values in R
+comment:  A practical demonstration of how missing values show up in R and how to deal with them. Note that this module does **not** cover statistical approaches for handling missing data, but instead focuses on the code you need to find, work with, and assign missing values in R.
+long_description: This is a beginners guide to handling missing values in R. It covers what `NA` values are and how to check for them, how to mark values as missing, how to work around missing values in your analysis, and how to filter your data to remove missingness. This module is appropriate for learners who feel comfortable with R basics and are ready to get into the realities of data analysis, like dealing with missing values.
+
+estimated_time: 45 min
+
+@learning_objectives  
+
+After completion of this module, learners will be able to:
+
+- check the number and location of missing values in a dataframe
+- mark values as missing
+- use common arguments like `na.rm` and `na.action` to control how functions handle missingness
+- remove cases with missing values from a dataframe
+
+@end
+
+script:  https://code.jquery.com/jquery-3.6.0.slim.min.js
+
+@gifPreload
+<script>
+(function($) {
+
+  // Get the .gif images from the "data-alt".
+	var getGif = function() {
+		var gif = [];
+		$('img').each(function() {
+			var data = $(this).data('alt');
+			gif.push(data);
+		});
+		return gif;
+	}
+
+	var gif = getGif();
+
+	// Preload all the gif images.
+	var image = [];
+
+	$.each(gif, function(index) {
+		image[index]     = new Image();
+		image[index].src = gif[index];
+	});
+
+	// Change the image to .gif when clicked and vice versa.
+	$('figure').on('click', function() {
+
+		var $this   = $(this),
+				$index  = $this.index(),
+
+				$img    = $this.children('img'),
+				$imgSrc = $img.attr('src'),
+				$imgAlt = $img.attr('data-alt'),
+				$imgExt = $imgAlt.split('.');
+
+		if($imgExt[1] === 'gif') {
+			$img.attr('src', $img.data('alt')).attr('data-alt', $imgSrc);
+		} else {
+			$img.attr('src', $imgAlt).attr('data-alt', $img.data('alt'));
+		}
+
+		// Add play class to help with the styling.
+		$this.toggleClass('play');
+
+	});
+
+})(jQuery);
+</script>
+@end
+
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+script: https://kit.fontawesome.com/83b2343bd4.js
+
+-->
+
+# @title
```

Comment:
```suggestion
Missing Values in R
```
Unfortunately, using `@title` causes the left TOC to break, IDK why...

Reply:
Ah, thank you! I keep forgetting that there's a good reason we don't use @title as a header...